frontend/src/common/PageWithTabs.tsx

```diff
@@ -90,10 +91,15 @@ export default function PageWithTabs({
           return (
             <Tab
               key={tab.title}
-              label={tab.title}
               disabled={tab.disabled}
               value={tab.path}
               data-test={`${tab.path}Tab`}
+              sx={{ pointerEvents: 'auto' }}
```

Comment:
Is this needed? I believe the default behaviour of `Tab` should handle this for us.